requirements/edx/base.txt

```diff
@@ -79,14 +79,14 @@ python-social-auth==0.2.12
 pytz==2015.2
 pysrt==0.4.7
 PyYAML==3.10
-requests==2.7.0
+requests==2.9.1
```

Comment:
@omarkhan I wanted to see the page that has the list of changes made to requests between version 2.7.0 and 2.9.1 just to be able to spot check that there were no changes that might break other functionality that relies on requests.  Same with stevedore below, since that is a big version jump.